mithril-common/src/chain_observer/cli_observer.rs

```diff
@@ -533,6 +576,14 @@ mod tests {
 
     use kes_summed_ed25519::{kes::Sum6Kes, traits::KesSk};
 
+    #[tokio::test]
+    async fn test_get_current_era() {
+        let observer = CardanoCliChainObserver::new(Box::<TestCliRunner>::default());
```

Comment:
I wonder if it's not better to pass the value `Conway` to the TestCliRunner  to know where it comes from.
It helps to understand the test but may add too much complexity to the TestCliRunner.

Reply:
It's updated in this [commit](https://github.com/input-output-hk/mithril/pull/2094/commits/a0299906f35dd5e469f9b6a2db737242e6d8403b).